integrations/dd-sdk-android-okhttp-otel/transitiveDependencies

```diff
@@ -2,13 +2,11 @@ Dependencies List
 
 com.squareup.okhttp3:okhttp:4.11.0                              :  768 Kb
```

Comment:
I've seen this particular change in a few prs unintentionally so just making sure -> in this pr we want this change?

Reply:
As I replied to Nikita above not sure why, need to check what's going on.